jdbc-driver/src/main/java/com/google/cloud/dataproc/DataprocDriver.java

```diff
@@ -0,0 +1,59 @@
+package com.google.cloud.dataproc;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.sql.*;
+import java.util.Properties;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+public class DataprocDriver implements Driver {
```

Comment:
Please add a Javadoc comment above every non-test Java class explaining what it is.